en/_posts/2017-01-31-kafka-metrics-grafana.md

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title:  "Display Kafka metrics using Grafana"
+categories: queue
+author: thcdrt
+lang: en
+---
+
+## Introduction
+
+This tutorial assumes you already have an OVH account, have subscribed to [OVH Kafka](https://www.runabove.com/dbaas-queue.xml) and have created an OVH Kafka application.
+
+If you don't, have a look at our [Getting Started](https://community.runabove.com/kb/en/queue/getting-started-with-queue-as-a-service.html) guide that will also introduce you to Kafka.
+
+This guide will help you to understand step by step how to display your Kafka applications metrics using [Grafana](https://grafana.tsaas.ovh.com) and the [Metrics Data Platform](https://www.ovh.com/fr/data-platforms/metrics/).
+
+## Preparation
+### Get your application token to use the Metrics API
+Read the token using this link: [Token OVH API](https://api.ovh.com/console/#/dbaas/queue/%7BserviceName%7D/metrics/account#GET).
```

Comment:
Now that we have a new improved version of the Sunrise Manager, the Metrics accessible is accessible directly on the manager "info" tab. I think it's more user-friendly to link and change the screenshot for the manager UI instead of the API console.
This way we can also remove the note related to the serviceName below, which can be a bit confusing.